lib/SampleService/core/validator/builtin.py

```diff
@@ -13,6 +13,7 @@
 If an exception is not thrown, and a falsy value is returned, the validation succeeds.
 '''
 
+import os
```

Comment:
Needs tests, something simple that uses the new unit should suffice

Reply:
I've added a very simple test, let me know if its not sufficient

Reply:
If you're going to allow the `cl` alias should add some tests for that as well, otherwise looks good